development_scripts/deconvolution_demo.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jun  5 14:43:48 2024
+
+Script demonstrating the mass transport deconvolution class & methods
+
+email a.winiwarter@imperial.ac.uk for the data
+
+@author: awiniwar
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+import ixdat
+
+from pathlib import Path
+from ixdat.techniques.deconvolution import ECMSImpulseResponse
+from ixdat import Measurement
+from ixdat.techniques.ms import MSCalResult
+from spectro_inlets_quantification import Calibration
+
+ixdat.config.plugins.activate_siq()
+
+# ------------ O2 calibration
+F_o2_mscal = MSCalResult(
+    name="O2", mol="O2", mass="M32", cal_type="ECMS calibration", F=0.11755
+)
+F_o2 = F_o2_mscal.to_siq()
+
+# ------------------ Impulse response fitting of measured data + comparison with model
+# insert your folder here:
+data_folder = Path(
+    r"C:\Users\awiniwar\OneDrive - Imperial College London\Data\photo ECMS\202405 data flurin+daniele\data_demo_script"
```

Comment:
Still an open problem, I guess, as to how can we best do this universally? The following now works for anyone with the dropbox folder in the normal place on their computer:
```
data_folder = Path.home() / "Dropbox/ixdat_resources/test_data/deconvolution"
```